### ISSUE TITLE
Require error?

### ISSUE THREAD

**Issue by austinhuang0131** [TooTallNate/node-bindings#39]
```
1|dtel     | Error: Module version mismatch. Expected 48, got 57.
1|dtel     |     at Error (native)
1|dtel     |     at Object.Module._extensions..node (module.js:597:18)
1|dtel     |     at Module.load (module.js:487:32)
1|dtel     |     at tryModuleLoad (module.js:446:12)
1|dtel     |     at Function.Module._load (module.js:438:3)
1|dtel     |     at Module.require (module.js:497:17)
1|dtel     |     at require (internal/module.js:20:19)
1|dtel     |     at bindings (/home/austin/node_modules/bindings/bindings.js:76:44)
1|dtel     |     at Object.<anonymous> (/home/austin/node_modules/node-opus/lib/OpusEncoder.js:4:34)
1|dtel     |     at Module._compile (module.js:570:32)
^C
[austin@runes ~]$ node -v
v8.0.0
[austin@runes ~]$ npm -v
5.0.0
```

**Comment by CyberiumShadow** [comment]
As stated in https://github.com/Rantanen/node-opus/issues/38
`npm install` should recompile any node modules you have that use native code.

**Comment by austinhuang0131** [comment]
Solved it like 1.5 months ago But forgot to close this lol
